### PR TITLE
Merge U-50 XRT driver changes

### PR DESCRIPTION
**Pull request by mfcpickett**
Add new U50 sensors

### CODE REVIEW

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c [hunk 1]
```diff
@@ -78,7 +78,14 @@
 #define	XMC_CAGE_TEMP3_REG		0x194
 #define	XMC_SNSR_CHKSUM_REG		0x1A4
 #define	XMC_SNSR_FLAGS_REG		0x1A8
-#define	XMC_HBM_TEMP_REG		0x260
+#define	XMC_HBM_TEMP1_REG		0x260
+#define XMC_VCC3V3_REG			0x26C
```
Comment (maxzhen) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c]: Indentation?

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c [hunk 2]
```diff
@@ -944,7 +1017,7 @@ static int get_temp_by_m_tag(struct xocl_xmc *xmc, char *m_tag)
 		return -ENODEV;
 
 	if (!strncmp(m_tag, "HBM", 3)) {
-		xmc_sensor(xmc->pdev, HBM_TEMP, &ret, SENSOR_INS);
+		xmc_sensor(xmc->pdev, HBM_TEMP1, &ret, SENSOR_INS);
```
Comment (maxzhen) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c]: All hbm banks has one sensor? then what is the usage for HBM_TEMP2?

**Comment by maxzhen** [comment]
ok to test

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c (hunk 2 above)
Comment (mfcpickett) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c]: My understanding of it is that the second sensor reading is used internally by the SC to calculate the temperature and by the end-user to check correct operation of the board.

Comment (maxzhen) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c]: I'm not following... if it's used internally by the SC, then please don't expose it. If it's used by end-user, then what is the difference b/w hbm-tmp1 and hbm-tmp2? the hbm-tmp1 is also used by end-user.

**Comment by mfcpickett** [comment]
Replaced by new PR

**Pull request closed without merging**